libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -2127,6 +2130,65 @@ void GCS_MAVLINK::send_raw_imu()
 #endif
 }
 
+#if AP_MAVLINK_MSG_HIGHRES_IMU_ENABLED
+void GCS_MAVLINK::send_highres_imu()
+{
+    uint16_t fields_updated = 0;
+    Vector3f accel;
+    Vector3f gyro;
```

Comment:
I know fields_updated declares the fields valid or not, but when then IMU is not available we should make sure we send 0s instead of random numbers off the stack. Same with mag.